flink-kubernetes/src/test/java/org/apache/flink/kubernetes/kubeclient/Fabric8FlinkKubeClientTest.java

```diff
@@ -411,6 +432,16 @@ void testStopAndCleanupCluster() throws Exception {
                 .isEmpty();
     }
 
+    @Test
+    void testWatchPodsAndDoCallback() throws Exception {
+        mockPodEventWithLabels(TESTING_LABELS);
+        // the count latch for events.
+        final CountDownLatch eventLatch = new CountDownLatch(4);
+        this.flinkKubeClient.watchPodsAndDoCallback(
+                TESTING_LABELS, new TestingKubernetesPodCallbackHandler(eventLatch));
+        assertTrue(eventLatch.await(10, TimeUnit.SECONDS));
```

Comment:
We need use junit5 and Assertj in unit tests. 

assertThat(eventLatch.await(1000, TimeUnit.MILLISECONDS)).isTrue();